storm-client/src/jvm/org/apache/storm/messaging/netty/Client.java

```diff
@@ -451,7 +451,6 @@ public int getPort() {
     public void close() {
         if (!closing) {
             LOG.info("closing Netty Client {}", dstAddressPrefixedName);
-            context.removeClient(dstHost, dstAddress.getPort());
```

Comment:
This line is the only reason to pass context into the Client.  Please remove all references to context if we are going to remove this.

Reply:
done